docs/guide/dev/code_documentation/README.md

```diff
@@ -64,29 +64,29 @@ This is an example of a class definition. It contains so many tags to show their
 
 	/**
 	 * Represents an editor instance. This constructor should be rarely
-	 * used, in favor of the {@linkapi CKEDITOR } editor creation functions.
+	 * used, in favor of the \{@linkapi CKEDITOR\} editor creation functions.
 	 *
 	 * ```js
-	 *	var editor = new {@linkapi CKEDITOR.editor CKEDITOR.editor}();
+	 *	var editor = new \{@linkapi CKEDITOR.editor CKEDITOR.editor\}();
```

Comment:
There are multiple occurences of code like `\{@linkapi CKEDITOR.editor CKEDITOR.editor\}` - it's a bad practice. Instead it should be simplified to `\{@linkapi CKEDITOR.editor\}`.